lib/ansible/modules/database/mysql/mysql_replication.py

```diff
@@ -103,6 +103,16 @@
         required: false
         default: null
         version_added: "2.0"
+    master_use_gtid:
```

Comment:
Actually, if we get this feature, the module should also support the remaining options (see https://mariadb.com/kb/en/library/change-master-to/):

- `IGNORE_SERVER_IDS`
- `DO_DOMAIN_IDS`
- `IGNORE_DOMAIN_IDS`

And if you can, `MASTER_SSL_VERIFY_SERVER_CERT = 0|1` (off-topic...).

Reply:
@AndiDog Maybe best to make your own PR with those required changes? Thanks!

Reply:
Sure, I can do that. Thought it might be related since those features are typically used together.

Reply:
Make a PR with everything that supersedes this one ?

Reply:
Or we can merge them ASAP and then you just add your stuff ?

Reply:
I'll wait until _this_ PR is finished – not in a hurry and no need for merge conflicts. Then I can add any further parameters.